### PR TITLE
Made make check pass on out-of-tree builds.

### PR DESCRIPTION
**Pull request by matejak**
The tests will pass when using `VPATH` build and with `sendmail` service not running.

### CODE REVIEW

**Comment by matejak** [comment]
Strangely enough, I have some issues running `make distcheck` locally.

File: tests/mitre/test_mitre.sh [hunk 1]
```diff
@@ -111,7 +113,11 @@ if [ $SELINUX_ENABLED -eq 0 ]; then
 	test_run "linux-def_selinuxsecuritycontext_test.xml" test_mitre linux-def_selinuxsecuritycontext_test.xml "true"
 fi
 
-test_run "linux-def_inetlisteningservers_test.xml" test_mitre linux-def_inetlisteningservers_test.xml "true"
+if  [ $SENDMAIL_RUNNING = yes ]; then
+	test_run "linux-def_inetlisteningservers_test.xml" test_mitre linux-def_inetlisteningservers_test.xml "true"
+else
+	echo "Skipped the inetlisteningservers test as sendmail service is not running." >&2
```
Comment (jan-cerny) [inline tests/mitre/test_mitre.sh]: I think we usually return 255 when we skip something, but I don't know if it's possible to use 255 here somehow

**Comment by mpreisler** [comment]
@matejak this needs a rebase

**Comment by mpreisler** [comment]
The jenkins fail looks legit:
```
/home/jenkins/workspace/openscap-pull-requests/openscap-1.2.17/_build/tests/offline_mode/../../../tests/offline_mode/test_offline_mode_textfilecontent54.sh: line 11: ../../../tests/offline_mode/../test_common.sh: No such file or directory
/home/jenkins/workspace/openscap-pull-requests/openscap-1.2.17/_build/tests/offline_mode/../../../tests/offline_mode/test_offline_mode_textfilecontent54.sh: line 54: test_init: command not found
FAIL: test_offline_mode_textfilecontent54.sh
```

**Comment by mpreisler** [comment]
@openscap-ci test this please

File: .gitignore [hunk 2]
```diff
@@ -4,7 +4,7 @@
 .gdb_history
 .libs/
 .*.swp
-Makefile
+Makefile$
```
Comment (mpreisler) [inline .gitignore]: what is the reason for this? no mention of it in the commit message

**Comment by mpreisler** [comment]
The Makefile thing is not a big deal...

ACK

File: .gitignore (hunk 2 above)
Comment (matejak) [reply .gitignore]: I am sorry about the missing message, it is a bugfix which prevents `Makefile.am` files to be git-ignored.

**Pull request merged**